src/ui/public/styles/variables/bootstrap-mods.less

```diff
@@ -46,7 +46,7 @@
 //** By default, this inherits from the `<body>`.
 @headings-font-family:    @font-family-base;
 @headings-font-weight:    400;
-@headings-line-height:    1.1;
+@headings-line-height:    1.3;
```

Comment:
Is there a reason why we have to set this explicitly instead of just letting the browser do its thing with `line-height: normal;`? This particular height might work with open sans, but I haven't tested it on every font in the font family on every OS in every browser.